spec/reset_password_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe 'password reset', type: :feature, js: true do
```

Comment:
All the tests in this repo will be feature specs.

That said, maybe it makes some sense to configure that assumption at the top level in `spec_helper.rb` so that we don't have to explicitly declare it in each file.